### PR TITLE
[ACS-8450] Fixed issue where required marker * was not showing up for create rule > Name field

### PR DESCRIPTION
**Pull request by swapnil-verma-gl**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
* marker was not showing up for rule name field

**What is the new behaviour?**
* marker now shows up for rule name field

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://hyland.atlassian.net/browse/ACS-8450

### CODE REVIEW

File: projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.scss [hunk 1]
```diff
@@ -33,6 +33,10 @@
       align-items: baseline;
       gap: 8px;
 
+      #{$mat-floating-label}#{$mat-floating-label-above} {
```
Comment (DenysVuika) [inline projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.scss:36]: this does not make sense, this is still indirect dependency on the material selectors imho

Comment (swapnil-verma-gl) [reply projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.scss]: Ive updated the selector to this - 
```
label[for="rule-details-name-input"] {
  transform: translateY(-20%) scale(1);
}
```
However, the strategy of using selectors via the mat-selectors.scss file is something that was done throughout the angular upgrade process. If we want to replace most/all of those occurences, then we should probably open create a ticket for that task

**Review by g-jaskowski — APPROVED** [review]

File: projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
     <div class="aca-rule-details__form__row">
       <label class="aca-rule-details__name__label" for="rule-details-name-input">{{ 'ACA_FOLDER_RULES.RULE_DETAILS.LABEL.NAME' | translate }}</label>
       <div>
-        <mat-form-field appearance="fill" floatLabel='auto' class="aca-rule-details__form__row__field">
+        <mat-form-field appearance="fill" floatLabel='always' class="aca-rule-details__form__row__field">
+          <mat-label *ngIf="name.value === ''">{{ 'ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.NAME' | translate }}</mat-label>
```
Comment (DenysVuika) [inline projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.html:8]: can be `*ngIf="!name.value"` to cover null/undefined as well

Comment (swapnil-verma-gl) [reply projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.html]: Added, thanks !

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**